bob.js

```diff
@@ -1142,6 +1156,19 @@ isILRace = () => {
     return categoryName === "Individual Levels";
 }
 
+//e.g. 1 --> "  1"
+addSpaces = (input, outputLength) => {
+    var spaceCount = outputLength - input.length;
+    if (spaceCount > 0) {
+        var spacesString = " ";
+        for (var i = 1; i < spaceCount; i++) {
```

Comment:
If this doesn't work I think you can use `let` instead of `var`, but I'm not sure if/what the difference is (not a javascript expert in the slightest)